test/e2e/k8s_util.go

```diff
@@ -285,7 +292,36 @@ func (k *KubernetesUtils) Probe(ns1, pod1, ns2, pod2 string, port int32, protoco
 		return Error, fmt.Errorf("no Pod of label pod=%s in Namespace %s found", pod2, ns2)
 	}
 	toPod := toPods[0]
+	fromPodName, toPodName := fmt.Sprintf("%s/%s", ns1, pod1), fmt.Sprintf("%s/%s", ns2, pod2)
+	return k.prodeAndDecideConnectivity(fromPod, toPod, fromPodName, toPodName, port, protocol)
+}
 
+func (k *KubernetesUtils) ProbeRemoteCluster(remoteCluster *KubernetesUtils,
```

Comment:
Better to converge this with `Probe` method, most codes are shared, you can check and set `toPods` when `remoteCluster` is not nil.

Reply:
Done